core/specfem/assembly/boundaries.cpp

```diff
@@ -2,3 +2,8 @@
 #include "boundaries/dim2/boundaries.tpp"
 #include "boundaries/dim2/impl/acoustic_free_surface.tpp"
 #include "boundaries/dim2/impl/stacey.tpp"
+// TODO: Implement dim3 boundaries - requires adapting to different mesh
```

Comment:
```suggestion
// TODO(Lucas : DIM3_BOUNDARIES_UPDATE): Implement dim3 boundaries - requires adapting to different mesh
```

Just to make it easier to `grep`